plugins/source/azure/docs/tables/azure_cdn_custom_domains.md

```diff
@@ -4,6 +4,8 @@
 
 The primary key for this table is **id**.
 
+## Relations
+This table depends on [azure_cdn_endpoints](azure_cdn_endpoints.md).
```

Comment:
Do we know what code change fixed this?

Reply:
Yeah, there was a function in the SDK to calculate the parents of tables that wasn't recursive before, but is now. I can find a link if you're interested